internal/renderers/skia/Cargo.toml

```diff
@@ -70,5 +68,8 @@ skia-safe = { version = "0.64", features = ["metal"] }
 [target.'cfg(not(any(target_os = "macos", target_family = "windows")))'.dependencies]
 skia-safe = { version = "0.64", features = ["gl"] }
 
+[target.'cfg(not(any(target_os = "macos", target_family = "windows", target_os = "android")))'.dependencies]
+winit = { version = "0.29.1-beta", optional = true, default-features = false }
```

Comment:
winit is only needed for `glutin::display::DisplayApiPreference::Glx(Box::new(winit::platform::x11::register_xlib_error_hook))`

So that is only valid if the user is using winit and with the same winit version.
For example, if we'd use skia with the Qt backend, that'd not work. 
So I think the proper thing to do before we make this renderer public is to let the user provide this callback.

Reply:
Shall I do this change?